draft-tschofenig-rats-psa-token.md

```diff
@@ -591,6 +594,27 @@ attestation tokens can wrap the serialised COSE_Sign1 or COSE_Mac0 in the media
 type defined in {{sec-iana-media-types}} or the CoAP Content-Format defined in
 {{sec-iana-coap-content-format}}.
 
+# Scalability Considerations
+{: #sec-scalability}
+
+IAKs can be either raw public keys or certified public keys.
+
+Certified public keys require the manufacturer to run the certification
+authority that issues x.509 certs for the IAKs.
+
+This approach provides sensibly better scalability properties compared to using
+raw public keys, namely:
+
+* storage requirements on the verifier side are minimised - the same
+  manufacturer's trust anchor is used for any number of devices,
+* the provisioning model is simpler and more robust since there is no need to
```

Comment:
The assumption is that the verifier gets the root CA (and maybe iCAs) via an endorsement (using a CoMID or otherwise).

Note that the exact split between pre-provisioned vs inlined endorsements is up to the manufacturer / deployment.  In that respect the `x5chain` is quite flexible: it can contain the EE cert only, EE + partial chain, EE + full chain up to the trust anchor (see [§2 of RFC9360](https://www.rfc-editor.org/rfc/rfc9360.html#section-2-5.4.1) for the details).

Constraints around network bandwidth and memory (esp. network buffers) are part of this tradeoff.

As far as CRLs, each cert should have the relevant endpoints advertised in a CDP extension.  Or CRLs should be periodically provisioned into the verifier.

But yes, the overhead is "manufacturers need to operate their own CA", which is probably not for everyone.

Reply:
an attempt at improving the prose in [52e3a43](https://github.com/thomas-fossati/draft-psa-token/pull/84/commits/52e3a4316a5c045b43bd2e111f4d521c37053917)

Reply:
LGTM